src/main/java/org/myrobotlab/service/config/DiyServoConfig.java

```diff
@@ -0,0 +1,15 @@
+package org.myrobotlab.service.config;
+
+import org.myrobotlab.framework.Plan;
+
+public class DiyServoConfig extends ServoConfig {
+
+  @Override
+  public Plan getDefault(Plan plan, String name) {
+    super.getDefault(plan, name);
+    addDefaultPeerConfig(plan, name, "motor", "MotorDualPwm");
```

Comment:
doesn't this need something like an encoder listener or a pin listener also?

Reply:
just the single listener could satisfy it 
if you want distinctly different ones, then you'll want to get into filtering it when serializing it back to saved config